working-groups.md

```diff
@@ -0,0 +1,65 @@
+# Working Groups 
+
+### [Business Planning][biz-wg]
+
+WG for performing feasibility study, and business planning to identify start up costs, fundraising mechanisms, and risks.
+
+📅 Weekly calls  
+🌱 Cultivators:   
+👪 Members: [**@business-planning-wg**][t-biz]  
+✅ Tasks: GitHub board > [`wg:business-planning` label][l-biz]  
+
+### [Finance][fin-wg]
+
+WG for establishing accounting practices, including setting up a bank account, administering payroll and expenses, invoicing, and handling remittances.
+
+📅 Weekly calls  
+🌱 Cultivators:  
+👪 Members: [**@finance-wg**][t-fin]  
+✅ Tasks: GitHub board > [`wg:finance` label][l-fin]  
+
+### [Governance][gov-wg]
+
+WG for drafting by-laws, articles of incorporation, and planning for the first General Meeting (and subsequent structure).
+
+📅 Biweekly calls  
+🌱 Cultivators: dc  
+👪 Members: [**@governance-wg**][t-gov]  
+✅ Tasks: GitHub board > [`wg:governance` label][l-gov]  
+
+### [Infrastructure][inf-wg]
+
+WG for setting up our "virtual office", in particular the infrastructure and processes to collaborate.
+
+📅 Biweekly(ish) calls  
+🌱 Cultivators: patcon & elon. (next rotation after..?)  
+👪 Members: [**@infra-wg**][t-inf]  
+✅ Tasks: GitHub board > [`wg:infra` label][l-inf]  
+
+### [Operations][ops-wg]
+
+WG for stewarding our shared practices and policies for effective remote collaboration.
+
+📅 We host biweekly calls  
+🌱 Cultivators:   
+👪 Members: [**@ops-wg**][t-ops]  
+✅ Tasks: GitHub board > [`wg:ops` label][l-ops]  
+
+<!-- Links: Working Groups -->
```

Comment:
Am I supposed to change all these shortlinks to fully written out? `https://link.hypha.coop/business-planning`

Reply:
no, I hope not! I see these more as internal, vs. ones used by people (@patcon tell me I'm wrong?)

Reply:
Either sounds fine! Don't feel need to change, but def no objection to it (i wasn't imagining we were deleting/changing these shortlinks tho, but more like append only)

Reply:
I don't think they are append only in general as I can imagine times when we want to retire things, but in this case I think we can add redundant, like:

- https://link.hypha.coop/biz-wg
- https://link.hypha.coop/business-planning -> biz-wg

Because it's hard to remember all the short forms, and whether it's biz-wg or wg-biz or biz...
Anyhow I don't use this, so leaving as is for now.

Reply:
Maybe we could worry about that when we near the 500 shortlink limit? (rebrandly keeps stats, so we can see what's unused when that time comes) There's just no need for that list to be clean or sorted imho

Reply:
resolving as it seems like not pertinent to this PR